src/c/examples/bitfields/res/configuration.yaml

```diff
@@ -6,21 +6,17 @@ Service:
   Port: 59999
   StartupMsg: Example bitfields device service started
 
-Clients:
-  core-metadata:
-    Host: localhost
-    Port: 59881
+# uncomment when running from command-line in hybrid mode without Config Provider
+#Clients:
+#  core-metadata:
+#    Host: localhost
+#    Port: 59881
```

Comment:
This is not needed if you also implement the -d/--dev flag to set the automatically set the different Hosts for hybrid mode. :-)
See these PRs in go.
https://github.com/edgexfoundry/go-mod-bootstrap/pull/516
https://github.com/edgexfoundry/go-mod-bootstrap/pull/541
https://github.com/edgexfoundry/go-mod-bootstrap/pull/601

Reply:
Thank you @lenny-intel. I have created an issue for this. https://github.com/edgexfoundry/device-sdk-c/issues/487